src/test/java/be/seeseemelk/mockbukkit/WorldMockTest.java

```diff
@@ -22,273 +13,277 @@
-	{
-		WorldMock world = new WorldMock();
-		assertEquals("Weather duration should be zero", 0, world.getThunderDuration());
-		assertFalse(world.isThundering());
-	}
-
-	@Test
-	public void setThunderDuration_AnyPositiveValue_ShouldBeThundering()
-	{
-		WorldMock world = new WorldMock();
-		int duration = 20;
-		world.setThunderDuration(duration);
-		assertEquals("Weather duration should be more than zero", duration, world.getThunderDuration());
-		assertTrue(world.isThundering());
-	}
-
-	@Test
-	public void setThundering_True_ThunderDurationShouldBePositive()
-	{
-		WorldMock world = new WorldMock();
-		world.setThundering(true);
-		assertTrue("Weather duration should be more than zero", world.getThunderDuration() > 0);
-		assertTrue(world.isThundering());
-	}
-
-	@Test
-	public void spawnZombieTest()
-	{
-		WorldMock world = new WorldMock();
-		Location location = new Location(world, 100, 20, 50);
-		Entity zombie = world.spawnEntity(location, EntityType.ZOMBIE);
-		assertEquals(100, zombie.getLocation().getBlockX());
-		assertEquals(20, zombie.getLocation().getBlockY());
-		assertEquals(50, zombie.getLocation().getBlockZ());
-		assertTrue(world.getEntities().size() > 0);
-	}
-
-	@Test
-	public void onCreated_TimeSetToBeZero()
-	{
-		WorldMock world = new WorldMock();
-		assertEquals("World time should be zero", 0L, world.getFullTime());
-		assertEquals("Day time should be zero", 0L, world.getTime());
-	}
-
-	@Test
-	public void setTime_DayTimeValue()
-	{
-		WorldMock world = new WorldMock();
-		world.setTime(20L);
-		assertEquals(20L, world.getTime());
-		assertEquals(20L, world.getFullTime());
-	}
-
-	@Test
-	public void setTime_FullTimeValue()
-	{
-		WorldMock world = new WorldMock();
-		world.setFullTime(3L * 24000L + 20L);
-		assertEquals(20L, world.getTime());
-		assertEquals(3L * 24000L + 20L, world.getFullTime());
-	}
-
-	@Test
-	public void setTime_FullTimeShouldBeAdjustedWithDayTime()
-	{
-		WorldMock world = new WorldMock();
-		world.setFullTime(3L * 24000L + 20L);
-		world.setTime(12000L);
-		assertEquals(12000L, world.getTime());
-		assertEquals(3L * 24000L + 12000L, world.getFullTime());
-	}
-
-	@Test
-	public void setTime_Event_Triggered()
-	{
-		WorldMock world = new WorldMock();
-		world.setTime(6000L);
-		world.setTime(10000L);
-		server.getPluginManager().assertEventFired(TimeSkipEvent.class, event -> 
-			event.getSkipAmount() == 4000L && event.getSkipReason().equals(TimeSkipEvent.SkipReason.CUSTOM));
-	}
-
-	@Test
-	public void onCreated_EnvironmentSetToBeNormal()
-	{
-		WorldMock world = new WorldMock();
-		assertEquals("World environment type should be normal", World.Environment.NORMAL, world.getEnvironment());
-	}
+    private ServerMock server;
+
+    @Before
+    public void setUp()
+    {
+        server = MockBukkit.mock();
+    }
+
+    @After
+    public void tearDown()
+    {
+        MockBukkit.unmock();
+    }
+
+    @Test
+    public void getBlockAt_StandardWorld_DefaultBlocks()
```

Comment:
Still not sure how to approach these snake + camelCase scenarios

Reply:
This is a certain method of naming your unit tests. It's essentially `method_scenario_result()`, but most pull requests don't follow this and I don't use it either anymore. I don't work in Java all that much anymore, so I don't have any specific preference on how these should be treated.